docs/template_plugin/tests/functional/op_reference/irdft.cpp

```diff
@@ -696,6 +771,45 @@ std::vector<IRDFTParams> generateParamsForIRDFT() {
                   expected_rdft3d_results_2,
                   op::v0::Constant::create<int64_t>(element::Type_t::i64, Shape{3}, {-3, -2, 2}),
                   op::v0::Constant::create<int64_t>(element::Type_t::i64, Shape{3}, {4, 5, 12})),
+
+       // irdft3d_reversed_axes
+       IRDFTParams(Shape{3, 4, 8, 2},
+                  Shape{4, 4, 8},
+                  ET,
+                  ET,
+                  input_data_7,
+                  expected_rdft3d_results_3,
+                  op::v0::Constant::create<int64_t>(element::Type_t::i64, Shape{3}, {2, 1, 0}),
+                  NULL),
+       // irdft3d_reversed_negative_axes
+       IRDFTParams(Shape{3, 4, 8, 2},
+                  Shape{4, 4, 8},
+                  ET,
+                  ET,
+                  input_data_7,
+                  expected_rdft3d_results_3,
+                  op::v0::Constant::create<int64_t>(element::Type_t::i64, Shape{3}, {-1, -2, -3}),
+                  NULL),
+
+        // irdft3d_reversed_axes_with_signals
+        IRDFTParams(Shape{3, 4, 8, 2},
+                    Shape{3, 3, 10},
```

Comment:
I think "expected_shape" should be {10, 3, 3} here, no?

Reply:
great catch! what's strange for me: expected shape is used only for test name, no for any assert - I'll try to confirm such behaviour